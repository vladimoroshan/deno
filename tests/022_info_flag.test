# The output assumes 003_relative_import.ts has already been run earlier
# and its output is cached to $DENO_DIR.
args: --info http://127.0.0.1:4545/tests/003_relative_import.ts
output: tests/022_info_flag.out
