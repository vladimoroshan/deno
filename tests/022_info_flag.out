local: [WILDCARD]deps/http/127.0.0.1_PORT4545/tests/003_relative_import.ts
type: TypeScript
compiled: [WILDCARD].js
map: [WILDCARD].js.map
deps: http://127.0.0.1:4545/tests/003_relative_import.ts
| http://127.0.0.1:4545/tests/subdir/print_hello.ts
